classes/Product.php

```diff
@@ -6150,13 +6160,23 @@ public static function addCustomizationPrice(&$products, &$customized_datas)
     }
 
     /**
+     * @deprecated since 8.0.0, the customization price impact is already included in Product::getPriceStatic.
+     *
      * Add customization price for a single product
      *
      * @param array $product Product data
      * @param array $customized_datas Customized data
      */
     public static function addProductCustomizationPrice(&$product, &$customized_datas)
     {
+        @trigger_error(
+            sprintf(
+                '%s is deprecated since version 8.0.0. The customization price impact is already included in Product::getPriceStatic.',
```

Comment:
```suggestion
                '%s is deprecated since version 9.0.0. The customization price impact is already included in Product::getPriceStatic.',
```